src/views/Login.js

```diff
@@ -90,20 +92,21 @@ export default class Login extends Component {
     if (user) {
       return (
         <div className="user-status">
-          <p className="msg">
-            Logged in as: <strong>{user.username}</strong> &nbsp;
-          </p>
           <UserSettings />
+          <p className="msg ">
+            Logged in as:{' '}
```

Comment:
should there be a name here?

Reply:
On line 98 there is the name. The "{' '}" is just providing a space between 'logged in' and the username